src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelInternal.java

```diff
@@ -500,12 +434,13 @@ private void checkValidation() {
     }
   }
 
-  OpenChannelRequest.OnErrorOption getOnErrorOption() {
-    return this.onErrorOption;
-  }
-
   /** Returns underlying channel's row buffer implementation. */
   RowBuffer<T> getRowBuffer() {
     return rowBuffer;
   }
+
+  /** Returns underlying channel's attributes. */
```

Comment:
This is only visible for tests right? So we could annotate it as `@VisibleForTests` to make it clear that it is not used in "production". WDYT?